crates/db_schema/src/impls/post.rs

```diff
@@ -535,6 +565,19 @@ mod tests {
       featured_local: false,
       url_content_type: None,
       scheduled_publish_time: None,
+      comments: 0,
+      controversy_rank: 0.0,
+      downvotes: 0,
+      upvotes: 0,
+      score: 0,
+      hot_rank: 0.0,
+      hot_rank_active: 0.0,
+      instance_id: InstanceId(0),
+      newest_comment_time: Utc::now(),
+      newest_comment_time_necro: Utc::now(),
+      report_count: 0,
+      scaled_rank: 0.0,
+      unresolved_report_count: 0,
     };
```

Comment:
Im not so happy with this approach for asserts, because you need to write out a lot of irrelevant stuff. Would be cleaner to make a helper function with individual asserts for each field.